ecosystem/resources/members/qiskit-nature-cp2k.toml

```
@@ -0,0 +1,8 @@
+name = "qiskit-nature-cp2k"
+url = "https://github.com/mrossinek/qiskit-nature-cp2k"
+description = "This project contains the utilities that are required for the communication between Qiskit Nature and CP2K."
+licence = "Apache License 2.0"
+contact_info = "_No response_"
+alternatives = "_No response_"
+affiliations = "_No response_"
+labels = [ "Chemistry",]
```

The reference paper was not carried over from the submitted issue form:
```suggestion
labels = [ "Chemistry",]
group = "applications"
reference_paper = "https://arxiv.org/abs/2404.18737"
```

EDIT: the `group` was also missing

---

Thanks! There was a bug in our automation but we've fixed it since